microprofile/fault-tolerance/src/main/java/io/helidon/microprofile/faulttolerance/CommandRetrier.java

```diff
@@ -273,4 +275,16 @@ private void updateMetricsAfter(Throwable cause) {
     private String createCommandKey() {
         return method.getName() + Objects.hash(context.getTarget(), context.getMethod().hashCode());
     }
+
+    /**
+     * Hystrix throws a {@code RuntimeException}, so we need to check
+     * the message to determine if it is a breaker exception.
+     *
+     * @param t Throwable to check.
+     * @return Outcome of test.
+     */
+    private static boolean isHystrixBreakerException(Throwable t) {
+        return t instanceof RuntimeException && t.getMessage().contains("Hystrix "
+                + "circuit short-circuited and is OPEN");
```

Comment:
Testing the contents of the exception message seems a bit fragile but there's probably no alternative. 

Is there an easy way to add a unit test that would fail if that message text were changed in some future release of Hystrix?

Reply:
Yes, this is quite ugly. Hystrix is no longer being developed and we're likely going to move away from it. Also, if that message changes, we should see plenty of unit and TCK failures :)